less/variables.less

```diff
@@ -0,0 +1,4 @@
+@tablet: ~"min-width: 576px";
```

Comment:
you probably don't have to set media breakpoints as bootstrap handles this with the sm/md/lg etc.